src/components/AppNavigation/AppNavigationHeader/AppNavigationHeaderDatePicker.vue

```diff
@@ -66,13 +68,15 @@ import formatDateRage from '../../../filters/dateRangeFormat.js'
 import DatePicker from '../../Shared/DatePicker.vue'
 import ChevronLeftIcon from 'vue-material-design-icons/ChevronLeft.vue'
 import ChevronRightIcon from 'vue-material-design-icons/ChevronRight.vue'
+import { NcButton as ButtonVue } from '@nextcloud/vue'
```

Comment:
Any specific reason you import them as ButtonVue? NcButton seems fine, no?

Reply:
I've done it for uniform reasons: in Mail App we are using NcButton as ButtonVue, no other reasons

Reply:
in mail we do that in some places when the button was renamed ButtonVue because was easier to replace it, but it should be just NcButton :)